src/packagedcode/nuget.py

```diff
@@ -12,10 +12,92 @@
 from packagedcode import models
 from packagedcode.utils import build_description
 
+
 """
 Handle NuGet packages and their manifests.
 """
-# TODO: add dependencies
+
+# TODO: Add section to handle dependencies of dependencies
+
+
+def get_dependencies(nuspec):
+    """
+    Return a list of Dependent package objects found in a NuGet `nuspec` object.
+    """
+    dependencies = []
+    try:
+        if "dependencies" in nuspec:
+            if "group" in nuspec["dependencies"]:
+                for group in nuspec["dependencies"]["group"]:
+                    if "dependency" in group:
+                        for dependency in group["dependency"]:
+                            #dependencies.append(dependency)
+                            dpurl = models.PackageURL(
+                            type='nuget',
+                            namespace=None,
+                            name=dependency.get("@id"),
+                            version=dependency.get("@version"),
+                            qualifiers=None
+                        )
+                            dep_pack = models.DependentPackage(
+                            purl=str(dpurl),
+                            extracted_requirement=dependency.get("@version"),
+                            scope="dependency",
+                            is_runtime=False,
+                            is_optional=False,
+                            is_resolved=True,
+                        )
+
+                            dependencies.append(dep_pack)
+                            
+            if "dependency" in nuspec.get("dependencies"):
```

Comment:
I would prefer something like this, and then reuse the `nuget_dependency` var below:
```suggestion
            nuget_dependencies = nuspec.get("dependencies") or {}
            nuget_dependency = nuget_dependencies.get("dependency")
            if nuget_dependency:
```

Reply:
Out of curiosity, any reason for `or {}` over `get("...", {})`?

Reply:
@jaitaiwan re:

> Out of curiosity, any reason for or {} over get("...", {})?

This is an idiom I like to use to make sure that I am getting a proper empty dict in all cases even if the get returns None or some emptyish value.
Getting an empty dict when you expect a dict enable chaining the next call without requiring an extra check for empties.

```Python
>>> repr({1: None, 2: "", 3: {"bar": "baz"}}.get(1, {}))
'None'
>>> repr({1: None, 2: "", 3: {"bar": "baz"}}.get(1, {}) or {})
'{}'
>>> repr({1: None, 2: "", 3: {"bar": "baz"}}.get(1) or {})
'{}'

```

Reply:
Fair call. thanks!